examples/seismic/acoustic/operators.py

```diff
@@ -51,6 +59,37 @@ def iso_stencil(field, m, s, damp, kernel, **kwargs):
     return [Eq(next, eq_time.subs({H: lap}))]
 
 
+def inject_src(src, field, s, model, kernel):
```

Comment:
it **might** be a symptom that we should start having different modules (files, functions, ... I don't know) for different kernels (OT2 vs OT4), rather than polluting most functions with if/then/else